### DISCUSSION TITLE
make sumplelogin.com public for old users

### DISCUSSION THREAD

**Discussion by 21Umi** [simple-login/app#1586] [Feature Requests]
Hello!
Most companies that respect themselves do not impose restrictions on existing users. Please bring back simplelogin.com and simplelogin.fr domains for old users, and apply changes only for new users.

Otherwise, the service simply loses any meaning in my eyes. Free domains are already few and far between.

**Comment by nguyenkims** [comment]
Hi due to an increasing numbers of abuses on some domains, we decide to temporarily limit their usage. Please rest assured that: 
- existing aliases created with these domains continue to work normally. The only limit is you can't create new aliases with these domains.
- we're going to add more domains. Btw if you have an idea of what domains that are easy to remember, not too expensive, please let us know by sending an email to hi[at]simplelogin.io
